### PR TITLE
[local-preview] remove the cleanup flag

### PR DESCRIPTION
**Pull request by Pothulapati**
## Description

This PR updates the run command to not have the `--rm` cleanup
flag and thereby allowing users to read the logs even after a
failure through the `docker container cp gitpod:logs.txt ./logs.txt`
command.

Signed-off-by: Tarun Pothulapati <tarun@gitpod.io>

## Related Issue(s)

Fixes https://github.com/gitpod-io/gitpod/issues/12655

## How to test
Preview at: [/docs/self-hosted/latest/local-preview
](https://deploy-preview-2693--gitpod-kumquat.netlify.app/docs/self-hosted/latest/local-preview
)
## Release Notes

```release-note
[local-preview] remove the cleanup flag
```

## Documentation

<a href="https://gitpod-staging.com/#https://github.com/gitpod-io/website/pull/2693"></a>

<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/2693"></a>

### CODE REVIEW

File: gitpod/docs/self-hosted/latest/local-preview.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ This install method runs a [K3s](https://k3s.io/) cluster inside a Docker contai
 Run the following command to get the `local-preview` Docker container up and running:
 
 ```bash
-docker run -p 443:443 --privileged --name gitpod --rm -it  --mount type=volume,source=gitpod,destination=/var/gitpod eu.gcr.io/gitpod-core-dev/build/local-preview
+docker run -p 443:443 --privileged --name gitpod -it  --mount type=volume,source=gitpod,destination=/var/gitpod eu.gcr.io/gitpod-core-dev/build/local-preview
```
Comment (Siddhant-K-code) [inline gitpod/docs/self-hosted/latest/local-preview.md:27]: is double space intended for docker command 🤔 before `--mount`? I think, it should be (doesn't matter though): 

```suggestion
docker run -p 443:443 --privileged --name gitpod -it --mount type=volume,source=gitpod,destination=/var/gitpod eu.gcr.io/gitpod-core-dev/build/local-preview
```

**Review by adrienthebo — APPROVED** [review]
LGTM, +/- @Siddhant-K-code's comment 😄

**Comment by Siddhant-K-code** [comment]
@Pothulapati I think, we can merge this after [this commit suggestion](https://github.com/gitpod-io/website/pull/2693#discussion_r965534711). Is there anything more to hold it?

**Comment by Pothulapati** [comment]
@Siddhant-K-code Ops! Missed updates on this! Updating!

**Review by Siddhant-K-code — APPROVED** [review]

**Pull request merged**